packages/react/src/Timeline/Timeline.module.css

```diff
@@ -0,0 +1,104 @@
+.Timeline {
+    display: flex;
+    flex-direction: column;
+
+    &:where([data-clip-sidebar]) {
+        .TimelineItem:first-child {
+            padding-top: 0;
+          }
+  
+        .TimelineItem:last-child {
+        padding-bottom: 0;
+        }
+    }
+}
+
+.TimelineItem {
+    position: relative;
+    display: flex;
+    padding: var(--base-size-16) 0;
+    margin-left: var(--base-size-16);
+
+    &::before {
+      position: absolute;
+      top: 0;
+      bottom: 0;
+      left: 0;
+      display: block;
+      width: 2px;
+      content: '';
+      /* stylelint-disable-next-line primer/colors */
+      background-color: var(--borderColor-muted);
+    }
+
+    &:where([data-condensed]){
+        padding-top: var(--base-size-4);
+        padding-bottom: 0;
+
+        &:last-child {
+          padding-bottom: var(--base-size-16);
+        }
+
+        .TimelineBadge {
+          height: 16px;
+          margin-top: var(--base-size-8);
+          margin-bottom: var(--base-size-8);
+          color: var(--fgColor-muted);
+          background-color: var(--bgColor-default);
+          border: 0;
+        }
+
+    }
+}
+
+.TimelineBadgeWrapper {
+    position: relative;
+    z-index: 1;
+}
+
+.TimelineBadge {
+    display: flex;
+    width: 32px;
+    height: 32px;
+    margin-right: var(--base-size-8);
+    /* stylelint-disable-next-line primer/spacing */
+    margin-left: -15px;
+    flex-shrink: 0;
+    overflow: hidden;
+    color: var(--fgColor-muted);
+    
+    /* TODOl not quite sure if this is the correct migration for this line */
+    background-color: var(--timelineBadge-bgColor);
```

Comment:
Can I get a check on this line? I wasn't quite sure what the correct mapping was here. i pulled the `--timelineBadge-bgColor` from the computer styles in production